servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/RuleDescriptionSerializer.java

```diff
@@ -261,6 +261,8 @@ private static RuleDescription parseFromEntry(Node xEntry) {
                                     case "Action":
                                         rd.setAction(parseActionFromElement(element));
                                         break;
+                                    default:
```

Comment:
[SpotBugs-P2]
This method contains a switch statement where default case is missing. Usually you need to provide a default case.